drivers/hmis/app/graphql/mutations/update_form_definition.rb

```diff
@@ -26,7 +26,7 @@ def resolve(id:, input:)
       # to match the expected format.
       # 2. The JSON Form Editor (old), which sends keys as a JSON string in the expected format and doesn't need
       # to be transformed, but calling recursively_transform on it is not harmful either.
-      definition.definition = recursively_transform(JSON.parse(input.definition))
+      definition.definition = recursively_transform(JSON.parse(input.definition)) if input.definition
```

Comment:
Makes sense. I realized that https://github.com/greenriver/hmis-warehouse/blob/ece60faeb16d063d3e150a0d4906d0ba796aa944/drivers/hmis/app/graphql/types/admin/form_definition_input.rb#L16 is unnecessary now so that can probably be removed just for clarity.